calyx-opt/src/passes/mod.rs

```diff
@@ -60,22 +51,15 @@ pub use data_path_infer::DataPathInfer;
 pub use dead_assignment_removal::DeadAssignmentRemoval;
 pub use dead_cell_removal::DeadCellRemoval;
 pub use dead_group_removal::DeadGroupRemoval;
-pub use discover_external::DiscoverExternal;
-pub use external_to_ref::ExternalToRef;
 pub use externalize::Externalize;
 pub use go_insertion::GoInsertion;
 pub use group_to_invoke::GroupToInvoke;
 pub use group_to_seq::GroupToSeq;
-pub use hole_inliner::HoleInliner;
 pub use infer_share::InferShare;
 pub use lower_guards::LowerGuards;
 pub use math_utilities::get_bit_width_from;
 pub use merge_assign::MergeAssign;
-pub use metadata_table_gen::Metadata;
 pub use papercut::Papercut;
-pub use par_to_seq::ParToSeq;
-pub use register_unsharing::RegisterUnsharing;
-pub use remove_ids::RemoveIds;
```

Comment:
I'm not sure if `remove_ids` is required by some other pass. It might be the case that `cell-sharing` generates IDs that need to be removed for other passes to function. @calebmkim can you remind us?

Reply:
iirc `remove-passes` is just to make the textual output nicer. The sharing pass generates IDs for the node each time the pass runs, within the sharing itself. (But they are removed in a separate pass: remove-ids).

Reply:
Does this mean we'd like to keep `remove_ids` in `passes`?

Reply:
Cool, thanks!